distribution/lib/Standard/Base/0.0.0-dev/src/Constants.enso

```diff
@@ -0,0 +1,3 @@
+## A value that marks the mode that allows column mismatches but reports them as
```

Comment:
Not only I don't like to `Constants` module, but also the description seems a bit off. 

> allows column

what is a column!?